docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -0,0 +1,91 @@
+# Required Software
+
+1. [Visual Studio 2015 with Update 1](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 1_.
+2. Visual Studio 2015 Extensibility Tools. If you already installed Visual Studio, choose "Modify" from the Programs and Features control panel, and check "Visual Studio Extensibility".
+
+# Getting the Code
+
+1. Clone https://github.com/dotnet/roslyn
+2. Run the "Developer Command Prompt for VS2015" from your start menu.
+3. Run `Restore.cmd` in the command prompt to restore NuGet packages.
+4. Due to [Issue #5876](https://github.com/dotnet/roslyn/issues/5876), you should build on the command line before opening in Visual Studio. Run `msbuild /v:m /m Roslyn.sln`
```

Comment:
The problem with switching the compiler version is there's no good way to do that _inside_ Visual Studio. We want to have a reasonable expectation that if I open Roslyn.sln and hit build that it does the same thing as MSBuild on the command line.

Talking with @agocke, his response was if you _do_ want to build the compiler with the compiler like cibuild, then you can just run cibuild.cmd and get that same experience. I agree that's not discoverable; @agocke or @jaredpar do we have better documents somewhere?